app/screens/channel_info/title/public_private/public_private.tsx

```diff
@@ -1,9 +1,18 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React from 'react';
-import {Text} from 'react-native';
+import {useManagedConfig} from '@mattermost/react-native-emm';
+import Clipboard from '@react-native-clipboard/clipboard';
+import React, {useCallback} from 'react';
+import {useIntl} from 'react-intl';
+import {StyleSheet, Text, View} from 'react-native';
+import {useSafeAreaInsets} from 'react-native-safe-area-context';
 
+import SlideUpPanelItem, {ITEM_HEIGHT} from '@app/components/slide_up_panel_item';
+import {SNACK_BAR_TYPE} from '@app/constants/snack_bar';
+import {bottomSheet, dismissBottomSheet} from '@app/screens/navigation';
+import {bottomSheetSnapPoint} from '@app/utils/helpers';
+import {showSnackBar} from '@app/utils/snack_bar';
```

Comment:
We import the files directly from the subfolder, not all the way up to the app. What does this means? Remove the `app/` from the imports. So...
`@app/components/...` => `@components/...`
`@app/constants/...` => `@constants/...`
etc...

Reply:
Done 👍🏻